source/user-manual/agents/remote-upgrading/create-custom-wpk/create-custom-wpk-automatically.rst

```diff
@@ -0,0 +1,75 @@
+.. Copyright (C) 2019 Wazuh, Inc.
+
+.. _create-custom-wpk-automatically:
+
+Automated custom WPK packages creation
+=======================================
+
+Wazuh provides an automated way of building WPK packages using docker so there is no need for any other dependency.
+
+To create a WPK package follow these steps:
+
+Requirements
+^^^^^^^^^^^^^
+
+ * Docker
+ * Git
+
+Download our wazuh-packages repository from GitHub and go to the wpk directory.
+
+ .. code-block:: console
+
+        $ git clone https://github.com/wazuh/wazuh-packages && cd wazuh-packages/wpk
+
+Execute the ``generate_wpk_package.sh`` script, with the different options you desire. This script will build a Docker image with all the necessary tools to create the WPK and run a container that will build it:
+
+ .. code-block:: console
+
+        $ ./generate_wpk_package.sh -h
+
+        Usage: ./generate_wpk_package.sh [OPTIONS]
+
+            -t,   --target-system <target>              [Required] Select target wpk to build [linux/windows]
+            -b,   --branch <branch>                     [Required] Select Git branch or tag e.g.
+            -d,   --destination <path>                  [Required] Set the destination path of package.
+            -k,   --key-dir <path>                      [Required] Set the WPK key path to sign package.
+            -a,   --architecture <arch>                 [Optional] Target architecture of the package [x86_64].
+            -j,   --jobs <number>                       [Optional] Number of parallel jobs when compiling.
+            -pd,  --package-directory <directory>       [Required for windows] Package name to pack on wpk.
+            -o,   --output <name>                       [Required] Name to the output package.
+            -c,   --checksum <path>                     [Optional] Generate checksum
+            -h,   --help                                Show this help.
+
+To use this tool the certificate and the key must be in the same directory.
```

Comment:
Please, indicate here what certificate is needed. 
"To use this tool, the previously required certificated (link to the parent page) and the key must be in the same directory"

Reply:
Done in https://github.com/wazuh/wazuh-documentation/pull/1308/commits/e89410a03a224177d448c2cb1b2aad358b372ed2